projects/hitl/software/tests/unit/test_can.py

```diff
@@ -44,8 +46,9 @@ def test_create_state_dictionary(can, logger):
 
 @pytest.mark.soft
 @pytest.mark.unit
-def test_set_and_get_state(can, logger):
+def test_set_and_get_state(can, can2, logger):
     logger.info("Testing set_state and get_state...")
 
     can.set_state("ThrottlePos", 50)
-    assert can.get_state("ThrottlePos") == 50
+    time.sleep(.1)
+    assert can2.get_state("ThrottlePos") == pytest.approx(50, rel=0.01) 
```

Comment:
add a comment explaining the need for pytest.approx here (rounding error)

OR...

We can reduce the resolution of the ThrottlePos message a little bit so 100% = 200, not 255, which gives us much nicer rounding (this would be a .dbc change)